ui/DesignSystem/Component/SourceBrowser/RevisionSelector.svelte

```diff
@@ -108,10 +131,11 @@
   hidden={expanded}>
   <div class="selector-avatar">
     <Avatar
-      title={revisions[0].identity.metadata.handle}
-      avatarFallback={revisions[0].identity.avatarFallback}
+      title={currentSelectedPeer.identity.metadata.handle}
```

Comment:
It's still `handle` :grimacing:

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/b7fe61d0bc3f0547a378b2c4d6b15657cb0ec679